modules/graphql/schema/types/bonus/fields/SellerBonusField.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types = 1);
+
+namespace app\modules\graphql\schema\types\bonus\fields;
+
+use app\components\helpers\ArrayHelper;
+use app\models\seller\wallet\seller_balance\SellersBalance;
+use app\modules\graphql\components\AuthHelper;
+use app\modules\graphql\components\BasePaginatedField;
+use app\modules\graphql\components\ResolveParameter;
+use app\modules\graphql\schema\types\bonus\SellerBonusType;
+
+/**
+ * Class SellerBonusField
+ */
+class SellerBonusField extends BasePaginatedField {
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function __construct() {
+		parent::__construct([
+			'name' => 'sellerBonus',
+			'description' => 'Бонусы продавца',
+			'type' => SellerBonusType::type(),
+		]);
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function resolve(ResolveParameter $resolveParameter):array {
+		$balance = SellersBalance::findOne(['user_id' => AuthHelper::authenticate()->id]);
+		return [
+			'sum' => ArrayHelper::getValue($balance, 'sum')
+		];
+	}
```

Comment:
```suggestion
	public static function resolve(ResolveParameter $resolveParameter):array {
		if (null === $balance = SellersBalance::findOne(['user_id' => AuthHelper::authenticate()->id])) return [];
		$resolveParameter->dataProvider = new ArrayDataProvider([
			'allModels' => [
				'sum' => ArrayHelper::getValue($balance, 'sum')
			]
		]);

		return $resolveParameter();
	}
```

В текущем варианте нет пагинации (если понадобится, а даже если нет - то пусть будет провайдер для однообразия кода) + лишняя операция на пустой выборке.

Обсуждаемо.